js/packages/web/src/views/home/index.tsx

```diff
@@ -150,6 +154,14 @@ export const HomeView = () => {
                     {liveAuctionsView}
                   </TabPane>
                   {auctionsEnded.length > 0 && (
+                  <TabPane
+                    tab={<span className="tab-title">Resales</span>}
+                    key={LiveAuctionViewState.Resale}
```

Comment:
Let's rename Resale to Marketplace...had second thoughts about the name.

Reply:
Maybe secondary